dolphinscheduler-ui/src/js/conf/home/router/module/datasource.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import i18n from '@/module/i18n/index.js'
```

Comment:
Is there a specific char between `i18n` and `from`? 
![image](https://user-images.githubusercontent.com/15820530/142348775-15080162-665b-4dfb-826a-eb2879a6ef50.png)

Reply:
It seem NIT here, this issue exists in the past, not related to this patch.